src/screens/LegacyAccountBackup.js

```diff
@@ -0,0 +1,160 @@
+// Copyright 2015-2019 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+'use strict';
+
+import React, { useEffect } from 'react';
+import { AppState, ScrollView, StyleSheet, Text, View } from 'react-native';
+import { Subscribe } from 'unstated';
+
+import colors from '../colors';
+import fonts from '../fonts';
+import fontStyles from '../fontStyles';
+import AccountCard from '../components/AccountCard';
+import Background from '../components/Background';
+import Button from '../components/Button';
+import ScreenHeading from '../components/ScreenHeading';
+import TouchableItem from '../components/TouchableItem';
+import DerivationPasswordVerify from '../components/DerivationPasswordVerify';
+import AccountsStore from '../stores/AccountsStore';
+import { NetworkProtocols, NETWORK_LIST } from '../constants';
+import { alertBackupDone, alertCopyBackupPhrase } from '../util/alertUtils';
+
+export default class LegacyAccountBackup extends React.PureComponent {
+	render() {
+		return (
+			<Subscribe to={[AccountsStore]}>
```

Comment:
why not use the HOC?

Reply:
Resolved, I have not check that, HOC is not 100% covered now.

Reply:
If we us HOC, most times `withNavigation` is used together. In this case need to make sure that the the wrapped component is not a Class but a function (`withNavigation` need that). Hence more work and test is needed, for 100% coverage and use HOC everywhere I suggest we could do that in the future PR.

Reply:
sounds good to me